### DISCUSSION TITLE
How to set per server avatars?

### DISCUSSION THREAD

**Discussion by Taimoor-Tariq** [discordjs/discord.js#6765] [Q&A] [answered]
Can someone help me in setting per server avatars

**Comment by almostSouji** [comment, marked as answer]
Setting per server avatars is not available to bots, just users with nitro.

**Reply by Taimoor-Tariq** [reply]
![image](https://user-images.githubusercontent.com/45585081/136060722-56461456-7753-446b-8835-4d31c62dc633.png)

**Reply by ImRodry** [reply]
What?

**Reply by BaumianerNiklas** [reply]
discord.js v13.2 just adds the ability to retrieve guild avatars from other `GuildMember`s, Discord provides no way for bots to set their own guild avatars
